api/src/main/java/com/codeforcommunity/dto/map/SiteFeatureProperties.java

```diff
@@ -45,22 +42,12 @@ public Boolean getTreePresent() {
     return treePresent;
   }
 
-  public Double getDiameter() {
+  public Double diameter() {
```

Comment:
What's the naming convention for this? (diameter vs getDiameter)

Reply:
Removed this from feature